src/test/java/bio/terra/service/snapshot/SnapshotServiceTest.java

```diff
@@ -598,6 +606,64 @@ void verifyPassportAuthNoConsentCode() {
     verifyNoInteractions(ecmService);
   }
 
+  @Test
```

Comment:
Can you add a positive test as well (approved, no created snapshot, no job)?

Reply:
yes! good idea